types/retentionoptions.pp

```diff
@@ -0,0 +1,7 @@
+type IcingaDB::RetentionOptions = Hash[
```

Comment:
can you please add a puppet-strings like comment:
```
# @summary this type blablabla something with retention blablabla
```

Reply:
Sure, no idea why I forgot that.